packages/blocks/src/api/validation/index.js

```diff
@@ -600,10 +601,11 @@ export function isEquivalentHTML( actual, expected, logger = createLogger() ) {
  * @param {Object}        attributes           Parsed block attributes.
  * @param {string}        originalBlockContent Original block content.
  * @param {Object}        logger           	   Validation logger object.
+ * @param {string}        mode           	   Validation mode.
  *
  * @return {Object} Whether block is valid and contains validation messages.
  */
-export function getBlockContentValidationResult( blockTypeOrName, attributes, originalBlockContent, logger = createQueuedLogger() ) {
+export function getBlockContentValidationResult( blockTypeOrName, attributes, originalBlockContent, logger = createQueuedLogger(), mode = ( get( global, [ 'wp', 'blockValidationMode' ], null ) || 'strict' ) ) {
```

Comment:
Oh maybe `wp.applyFilters` as @dmsnell suggested instead of an obvious global. I'll try doing that tomorrow.

Reply:
Thanks @dmsnell, this is looking a little saner for the signature https://github.com/WordPress/gutenberg/pull/19188/commits/fea65b9f71384cd50ef2a467d06d0b623c98be7a